kmod/src/inode.c

```diff
@@ -1452,6 +1495,9 @@ struct inode *scoutfs_new_inode(struct super_block *sb, struct inode *dir,
 	scoutfs_lock_add_coverage(sb, lock, &si->ino_lock_cov);
 	si->drop_invalidated = false;
 	si->flags = 0;
+	si->worm_bits = SCOUTFS_WORM_V1_BIT;
```

Comment:
New inodes should have the bits initialized to 0 so they're not trying to enforce an ancient earliest time :).